src/main/kotlin/com/avail/compiler/splitter/MessageSplitter.kt

```diff
@@ -119,17 +153,16 @@ class MessageSplitter
 	 *  * Operator characters are never beside spaces, and are always parsed as
 	 *    individual tokens.
 	 *  * [Open guillemet][Metacharacter.OPEN_GUILLEMET] («),
-	 *    [double dagger][Metacharacter.DOUBLE_DAGGER] (‡), and [close
-	 *    guillemet][Metacharacter.CLOSE_GUILLEMET] (») are used to indicate
-	 *    repeated or optional substructures.
+	 *    [double dagger][Metacharacter.DOUBLE_DAGGER] (‡), and
+	 *    [close&#32;guillemet][Metacharacter.CLOSE_GUILLEMET] (») are used to
+	 *    indicate repeated or optional substructures.
 	 *  * The characters [octothorp][Metacharacter.OCTOTHORP]
 	 *    (#) and [question mark][Metacharacter.QUESTION_MARK] (?) modify the
 	 *    output of repeated substructures to produce either a count of the
 	 *    repetitions or a boolean indicating whether an optional subexpression
 	 *    (expecting no arguments) was present.
 	 *  * Placing a [question mark][Metacharacter.QUESTION_MARK]
-	 *    (?) after a group containing arguments but no [double
-	 *    dagger][Metacharacter.DOUBLE_DAGGER] (‡) will limit the repetitions of
+	 *    (?) after a group containing arguments but no [double&#32;dagger][Metacharacter.DOUBLE_DAGGER] (‡) will limit the repetitions of
```

Comment:
Long line.

Reply:
There were actually two long lines in this file. I fixed both.